File: PRAKTIKUM MODUL 1 C DAN PY/BAHASA C/PRAK108-2610817210007-MuhammadZakiMubarak.c
#include <stdio.h>
#define _USE_MATH_DEFINES
#include <math.h>


int main() {
    float turn = 5.0;
    float distance = 14.0;
    float circumference = distance / turn;
    float radius = circumference / (2 * M_PI);

    printf("diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %.0f putaran\n", turn);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n\n", distance);
    printf("jawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", radius);

    return 0;
}

File: PRAKTIKUM MODUL 1 C DAN PY/PYTHON/PRAK108-2610817210007-MuhammadZakiMubarak..py
import math

turn = 5
distance_km = 14.0
around = distance_km / turn
radius = around / (2 * math.pi)

print("diketahui :")
print("Pak Dengklek mengelilingi taman = %d putaran" % turn)
print("Jarak tempuh Pak Dengklek = %.0f Kilometer" % distance_km)
print("\nJawaban :")
print("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer" % radius)